src/main/java/redis/clients/jedis/JedisClusterInfoCache.java

```diff
@@ -61,19 +62,35 @@ public void run() {
   }
 
   public JedisClusterInfoCache(final JedisClientConfig clientConfig, final Set<HostAndPort> startNodes) {
-    this(clientConfig, null, startNodes);
+    this(clientConfig, null, null, startNodes);
+  }
+
+  public JedisClusterInfoCache(final JedisClientConfig clientConfig, ClientSideCache csCache, final Set<HostAndPort> startNodes) {
```

Comment:
For consistency and alignment multiple constructors should follow an additive pattern. This means, adding the piece (csCache) as the last variable in this new constructor.